### ISSUE TITLE
Smart sync

### ISSUE THREAD

**Issue by LukeShortCloud** [ekultails/mcram#2]
Although rsync will already only sync up files from the RAM mount if they are changed, Minecraft updates its own server data files even if nothing has changed. There should be a "smart search" feature that will look through the detached "mcrsync" screen to tell if any users are currently logged in (i.e. do a grep count for the number of times a person has logged in and out; if the logged in status is higher than logged out that means they are logged in). If none are logged in, it should sync once and then stop until another person joins. This will further reduce the amount of writes to the drive.

tl;dr
Feature request: stop syncing once users log out.
